odoo/addons/base/tests/test_module.py

```diff
@@ -44,7 +44,7 @@ def test_default_manifest(self):
             'data': [],
             'demo': [],
             'demo_xml': [],
-            'depends': [],
+            'depends': ['base'],
```

Comment:
Do you mean every module implicitly **directly** depends on `base`?
It will unnecessarily increase the number of edges of the graph.
And I prefer to have everything explicit when I get a pre-processed manifest data.

I choose the idea:
if the `depends` is not specified or falsy, it depends on `base`

Reply:
No sure we need to be so strict.  This would be fine to me:
* if no `depends` (empty list or unspecified), then use `depends=['base']` instead (except for module `base`)
* otherwise use the given list